forum/users/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Role
+from django.contrib.auth.forms import AuthenticationForm
+
+class UserLoginForm(AuthenticationForm):
+    username = forms.EmailField(label="Email")
+    password = forms.CharField(widget=forms.PasswordInput, label="Password")
```

Comment:
You could improve the user experience by adding placeholders or help texts to the form fields.
```
username = forms.EmailField(
    label="Email", 
    widget=forms.EmailInput(attrs={
        'placeholder': 'Enter your email address',
        'class': 'form-control'
    }),
    help_text="Please enter the email associated with your account."
)
```